libcxxabi/src/demangle/ItaniumDemangle.h

```diff
@@ -4330,9 +4330,12 @@ Node *AbstractManglingParser<Derived, Alloc>::parseType() {
     case 'h':
       First += 2;
       return make<NameType>("half");
-    //                ::= DF <number> _ # ISO/IEC TS 18661 binary floating point (N bits)
+    //       ::= DF16b         # C++23 std::bfloat16_t
+    //       ::= DF <number> _ # ISO/IEC TS 18661 binary floating point (N bits)
     case 'F': {
       First += 2;
+      if (consumeIf("16b"))
+        return make<NameType>("bfloat16_t");
```

Comment:
Should this be `std::bfloat16_t` instead?

Reply:
I was wondering that. I have a feeling it should be `std::bfloat16_t` but I don't feel I can rule either way.

It's commented as `std::bfloat16_t` [here](https://github.com/itanium-cxx-abi/cxx-abi/blob/main/abi.html#L5206), for what it's worth.

Reply:
Also, out of curiosity, could/would it be  `std::bfloat16_t` in libcxxabi and `bfloat16_t` in LLVM? Or `std::` across the board? Should the two ever differ?

Reply:
I've updated the PR to print `std::bfloat16_t` in both cases